### PR TITLE
Add create system command

### PR DESCRIPTION
**Pull request by clebs**
**Description**

Changes proposed in this pull request:

- Add create system command to connect external apps.
- Updated various dependencies.
- The command outputs a token for the external system to connect.

Standard Output:

YAML output:

JSON output:

**Related issue(s)**
#554

### CODE REVIEW

File: cmd/kyma/create/system/system.go [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+		Use:   "system",
+		Short: "Creates a system on the Kyma cluster.",
+		Long: `Use this command to create a system on a Kyma cluster.
+
+### Detailed description
+
+A system in Kyma is used to bind external systems and applications into the cluster and allow Kyma services and functions to receive events and communicate with them.
+Once a system is created in kyma, use the token provided by this command to grant access to the external system into the kyma cluster.
+
+To generate a new token, rerun the same command with the --update flag.
```
Comment (bszwarc) [inline cmd/kyma/create/system/system.go:45]:
```suggestion
A system in Kyma is used to bind external systems and applications to the cluster and allow Kyma services and functions to communicate with them and receive events.
Once a system is created in Kyma, use the token provided by this command to allow the external system to access the Kyma cluster.

To generate a new token, rerun the same command with the ` + "`--update`" + ` flag.
```

File: cmd/kyma/create/system/system.go [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+		RunE:    func(_ *cobra.Command, args []string) error { return c.Run(args) },
+		Aliases: []string{"sys"},
+	}
+
+	cmd.Args = cobra.ExactArgs(1)
+
+	cmd.Flags().StringVarP(&o.Namespace, "namespace", "n", "", "Namespace to bind the system to.")
+	cmd.Flags().BoolVarP(&o.Update, "update", "u", false, "Update an existing system and/or generate a new token for it.")
+	cmd.Flags().StringVarP(&o.OutputFormat, "output", "o", "", "Specify the format of the output of the command. Supported formats: yaml, json.")
+	cmd.Flags().DurationVarP(&o.Timeout, "timeout", "", 2*time.Minute, "Time-out after which CLI stops watching the installation progress.")
```
Comment (bszwarc) [inline cmd/kyma/create/system/system.go:57]:
```suggestion
	cmd.Flags().DurationVarP(&o.Timeout, "timeout", "", 2*time.Minute, "Timeout after which CLI stops watching the installation progress.")
```

File: cmd/kyma/create/system/system.go [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+`,
+		RunE:    func(_ *cobra.Command, args []string) error { return c.Run(args) },
+		Aliases: []string{"sys"},
+	}
+
+	cmd.Args = cobra.ExactArgs(1)
+
+	cmd.Flags().StringVarP(&o.Namespace, "namespace", "n", "", "Namespace to bind the system to.")
+	cmd.Flags().BoolVarP(&o.Update, "update", "u", false, "Update an existing system and/or generate a new token for it.")
+	cmd.Flags().StringVarP(&o.OutputFormat, "output", "o", "", "Specify the format of the output of the command. Supported formats: yaml, json.")
```
Comment (bszwarc) [inline cmd/kyma/create/system/system.go:56]:
```suggestion
	cmd.Flags().StringVarP(&o.OutputFormat, "output", "o", "", "Specifies the format of the command output. Supported formats: YAML, JSON.")
```

File: cmd/kyma/create/system/system.go [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+
+`,
+		RunE:    func(_ *cobra.Command, args []string) error { return c.Run(args) },
+		Aliases: []string{"sys"},
+	}
+
+	cmd.Args = cobra.ExactArgs(1)
+
+	cmd.Flags().StringVarP(&o.Namespace, "namespace", "n", "", "Namespace to bind the system to.")
+	cmd.Flags().BoolVarP(&o.Update, "update", "u", false, "Update an existing system and/or generate a new token for it.")
```
Comment (bszwarc) [inline cmd/kyma/create/system/system.go:55]:
```suggestion
	cmd.Flags().BoolVarP(&o.Update, "update", "u", false, "Updates an existing system and/or generates a new token for it.")
```

**Review by bszwarc — COMMENTED** [review]
Added review :)

**Review by bszwarc — APPROVED** [review]

**Review by shorim — APPROVED** [review]

**Pull request merged**